spec/features/interactions/standard_interaction_spec.rb

```diff
@@ -0,0 +1,66 @@
+require 'rails_helper'
+require './db/sample_data/users'
+require './db/sample_data'
+
+RSpec::Steps.steps "Demo" do
+  it 'loads the seeds and populate with sample data' do
+    Rails.application.load_seed
+    add_sample_data
+  end
+
+  it "should show the welcome page on startup" do
+    visit root_path
+    expect(page).to have_text I18n.t('start_page.welcome_to')
+  end
+
+  it 'should let an organizer log in' do
+    #TODO: Use id instead of visible string, currently id is missing for login button
+    click_link I18n.t('navbar.login')
+    login_with_credentials(user_organizer.email, user_password)
+    expect(page).to_not have_text I18n.t('devise.failure.inactive')
+    expect(page).to_not have_text I18n.t('devise.failure.invalid')
+    expect(page).to_not have_text I18n.t('devise.failure.not_found_in_database')
+    expect(page).to_not have_text I18n.t('devise.failure.locked')
+    expect(page).to have_text I18n.t('start_page.welcome_to')
+  end
+
+  it 'should let organizer click the new event button' do
+    click_link I18n.t('navbar.events')
+    click_link 'new_event'
+  end
+
+  it 'should let organizer fill out event creation page' do
+    #TODO: Add radio button choice
```

Comment:
That's right, I forgot to mention it. I think it is impossible to select them with capybara in their current implementation. Since the default values correlate with the stories needs I left it out first while seeking for help now.

@nstrelow From what I saw, these are two radio buttons that don't get clicked, instead their class changes from `btn btn-default` to `btn ban-default active`, right? Do you have any idea on how to get this accomplished in capybara in a beautiful way?